packages/fxa-admin-panel/src/components/AccountSearch/Account/index.test.tsx

```diff
@@ -208,6 +209,9 @@ it('displays the account', async () => {
   expect(getByTestId('account-created-at')).toHaveTextContent(
     accountResponse.createdAt.toString()
   );
+  expect(getByTestId('account-locale')).toHaveTextContent(
+    accountResponse.locale || ''
```

Comment:
Why the `|| ''`?  When the test pass how do I tell which value was the actual value that caused it to pass?

Reply:
Ah this was just cause a type check would fail otherwise. Thanks for call it out. I did this quickly and forgot to circle back. I've switched to use the non-null assertion operator. To the know true value, you just have to look at the definition of accountResponse.